### PR TITLE
Readme file updated

### PR DESCRIPTION
**Pull request by manojsalunke85**
Signed-off-by: manoj <manoj.salunke@intel.com>

### CODE REVIEW

File: tests/automation_framework/README.md [hunk 1]
```diff
@@ -6,13 +6,33 @@ This is to validate the response of worker, work order and work
...
+   Edit env.py and change the test_mode to "listener"
+   Command: pytest -m listener
+   
+   SDK Mode:
+   Edit env.py and change the test_mode to "sdk"
+   Command: pytest -m sdk
+   
+   Proxy(Ethereum) Mode:
+   Edit env.py and change the blockchain_type to "ethereum", test_mode to "sdk", proxy_mode to "True"
+   Command: pytest -m ethereum
```
Comment (anjalirai-intel) [inline tests/automation_framework/README.md]: This marker will not exist in future

File: tests/automation_framework/README.md [hunk 2]
```diff
@@ -6,13 +6,33 @@ This is to validate the response of worker, work order and work
...
+2. Open a Docker container shell using following command. 
+     docker exec -it avalon-shell bash
+4. Running automation tests in Listener/SDK/Proxy Mode(Ethereum/Fabric) using markers.
+   Listener Mode:
+   cd $TCF_HOME/tests/automation_framework
+   Edit env.py and change the test_mode to "listener"
+   Command: pytest -m listener
+   
+   SDK Mode:
+   Edit env.py and change the test_mode to "sdk"
```
Comment (jinengandhi-intel) [inline tests/automation_framework/README.md]: Mention to leave the proxy mode to False.

File: tests/automation_framework/README.md [hunk 3]
```diff
@@ -6,13 +6,33 @@ This is to validate the response of worker, work order and work
...
-	docker exec -it tcf bash
-3. Run automation
-	cd $TCF_HOME/tests/validation_suite
-	pytest (All tests)
-	pytest work_order_tests/test_work_order_success.py (single test)
+1. Build Avalon: Refer to Build.md from the link below.
+     https://github.com/hyperledger/avalon/blob/master/BUILD.md
+2. Open a Docker container shell using following command. 
+     docker exec -it avalon-shell bash
+4. Running automation tests in Listener/SDK/Proxy Mode(Ethereum/Fabric) using markers.
```
Comment (jinengandhi-intel) [inline tests/automation_framework/README.md]: Add a new line before the Listener mode starts.

File: tests/automation_framework/README.md [hunk 4]
```diff
@@ -6,13 +6,33 @@ This is to validate the response of worker, work order and work
...
+   Command: pytest -m sdk
+   
+   Proxy(Ethereum) Mode:
+   Edit env.py and change the blockchain_type to "ethereum", test_mode to "sdk", proxy_mode to "True"
+   Command: pytest -m ethereum
+   
+   Proxy(Fabric) Mode:
+   Edit env.py and change the blockchain_type to "fabric", test_mode to "sdk", proxy_mode to "True"
+   Command: pytest -m fabric
+5. To run single test
```
Comment (jinengandhi-intel) [inline tests/automation_framework/README.md]: Add a new line before point 5

File: tests/automation_framework/README.md [hunk 5]
```diff
@@ -6,13 +6,33 @@ This is to validate the response of worker, work order and work
...
+	cd $TCF_HOME/tests/automation_framework
+	e.g pytest -k "test_workordersumit_success"
+6. To run all the tests from specific APIs (e.g Worker Setstatus/Worker Register/Workorder Submit)
```
Comment (jinengandhi-intel) [inline tests/automation_framework/README.md]: Add a new line after point 5.

**Review by jinengandhi-intel — CHANGES_REQUESTED** [review]

File: tests/automation_framework/README.md (hunk 1 above)
Comment (manojsalunke85) [reply tests/automation_framework/README.md]: changed it to proxy

File: tests/automation_framework/README.md (hunk 3 above)
Comment (manojsalunke85) [reply tests/automation_framework/README.md]: done

File: tests/automation_framework/README.md (hunk 4 above)
Comment (manojsalunke85) [reply tests/automation_framework/README.md]: done

File: tests/automation_framework/README.md (hunk 5 above)
Comment (manojsalunke85) [reply tests/automation_framework/README.md]: done

**Comment by anjalirai-intel** [comment]
Rebase required because of conflicting changes

**Pull request closed without merging**